scripts/nb-tester/qiskit_docs_notebook_tester/__init__.py

```diff
@@ -24,27 +25,16 @@
 from qiskit_ibm_runtime import QiskitRuntimeService
 from squeaky import clean_notebook
 
-NOTEBOOKS_GLOB = "[!.]*/**/*.ipynb"
-NOTEBOOKS_EXCLUDE = [
-    "docs/api/**",
-    "**/.ipynb_checkpoints/**",
-]
-NOTEBOOKS_THAT_SUBMIT_JOBS = [
-    "docs/start/hello-world.ipynb",
-    "tutorials/build-repitition-codes/notebook.ipynb",
-    "tutorials/chsh-inequality/notebook.ipynb",
-    "tutorials/grovers-algorithm/notebook.ipynb",
-    "tutorials/quantum-approximate-optimization-algorithm/notebook.ipynb",
-    "tutorials/repeat-until-success/notebook.ipynb",
-    "tutorials/submitting-transpiled-circuits/notebook.ipynb",
-    "tutorials/variational-quantum-eigensolver/notebook.ipynb",
-]
-
+def get_globs(path: str) -> dict[str, str | list[str]]:
```

Comment:
Nit: more appropriate name would be `read_config`. It's not quite globs, which I would expect to be a `list[str]`.

Consider used `TypedDict` to make this interface more clear: https://docs.python.org/3/library/typing.html#typing.TypedDict. I recommend it - it makes the code much more obvious.

At that point, I like to use this idiom:

```python
from __future__ import annotations

from dataclasses import dataclass

@dataclass
class Config:
    opt1: str
    opt2: bool

     @classmethod
     def read(cls, path: read) -> Config:
         return cls(...)
```

`config = Config.read()` reads quite nicely.

Reply:
Added benefits of the dataclass:

1. You can error if required keys are not set
2. Avoid `KeyError` when accessing the config. Right now, `globs["notebooks-that-submit-jobs"])` would be a cryptic error if it was not set properly in the TOML file.

Reply:
Nice, thanks!